### ISSUE TITLE
Unable to start second batch of training - im2txt model

### ISSUE THREAD

**Issue by cescamilla** [tensorflow/models#642]
After an initial training of 10k steps, i'm unable to start a new batch of training because no matter the number of steps i set, it only runs one, i faced this behaviour before in a Mac laptop with a GT 750M, Tensorflow 11 and Cuda 8.0

Ubuntu 14.04 Core i7 and 16 GB RAM
Tensorflow r0.11 
(https://storage.googleapis.com/tensorflow/linux/gpu/tensorflow-0.11.0-cp27-none-linux_x86_64.whl)
CUDA 8.0
cudNN 5.1
Graphic card: GTX 1080

command:
``
bazel-bin/im2txt/train --input_file_pattern="/home/ooyala/im2txt/data/mscoco/train-?????-of-00256" --inception_checkpoint_file="/home/ooyala/im2txt/data/inception_v3.ckpt" --train_dir="/home/ooyala/im2txt/model/train" --train_inception=false --number_of_steps=10000
``

output:
``
I tensorflow/stream_executor/dso_loader.cc:111] successfully opened CUDA library libcublas.so locally
I tensorflow/stream_executor/dso_loader.cc:111] successfully opened CUDA library libcudnn.so locally
I tensorflow/stream_executor/dso_loader.cc:111] successfully opened CUDA library libcufft.so locally
I tensorflow/stream_executor/dso_loader.cc:111] successfully opened CUDA library libcuda.so.1 locally
I tensorflow/stream_executor/dso_loader.cc:111] successfully opened CUDA library libcurand.so locally
INFO:tensorflow:Prefetching values from 256 files matching /home/ooyala/im2txt/data/mscoco/train-?????-of-00256
I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:925] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
I tensorflow/core/common_runtime/gpu/gpu_device.cc:951] Found device 0 with properties: 
name: GeForce GTX 1080
major: 6 minor: 1 memoryClockRate (GHz) 1.7335
pciBusID 0000:01:00.0
Total memory: 7.92GiB
Free memory: 7.63GiB
I tensorflow/core/common_runtime/gpu/gpu_device.cc:972] DMA: 0 
I tensorflow/core/common_runtime/gpu/gpu_device.cc:982] 0:   Y 
I tensorflow/core/common_runtime/gpu/gpu_device.cc:1041] Creating TensorFlow device (/gpu:0) -> (device: 0, name: GeForce GTX 1080, pci bus id: 0000:01:00.0)
INFO:tensorflow:Starting Session.
INFO:tensorflow:Starting Queues.
INFO:tensorflow:global_step/sec: 0
INFO:tensorflow:global step 10002: loss = 2.7211 (11.57 sec/step)
INFO:tensorflow:Stopping Training.
INFO:tensorflow:Finished training! Saving model to disk.
``

**Comment by cescamilla** [comment]
setting num_of_steps > my last number of steps, works. like pagination, i did not see that in the documentation.
